packages/hippodrome-ui/src/layouts/Layout.tsx

```diff
@@ -104,12 +107,13 @@ const Layout: React.FC = ({ children }) => {
       </HStack>
 
       {!address && <SplashScreen />}
-      {address && children}
+      {address && supportedNetworks.includes(chain.chainId!) && children}
+      {address && !supportedNetworks.includes(chain.chainId!) && <WrongNetwork />}
 
       <VStack mt="10" spacing="8">
         <Heading size="sm">Powered by</Heading>
         <HStack spacing="8">
-          <a href="https://arena.cryptocolosseum.com" target="_blank">
+          <a href="https://arena.cryptocolosseum.com" target="_blank" rel="noreferrer">
```

Comment:
Adding no `noreferrer` here means we won't show in google analytics as to the referrals. @tobowers